gobblin-utility/src/main/java/org/apache/gobblin/util/jdbc/DataSourceProvider.java

```diff
@@ -51,6 +57,15 @@ public class DataSourceProvider implements Provider<DataSource> {
   public DataSourceProvider(@Named("dataSourceProperties") Properties properties) {
     this.basicDataSource = new BasicDataSource();
     this.basicDataSource.setDriverClassName(properties.getProperty(CONN_DRIVER, DEFAULT_CONN_DRIVER));
+    // the validation query should work beyond mysql; still, to bypass for any reason, heed directive
+    if (!Boolean.parseBoolean(properties.getProperty(SKIP_VALIDATION_QUERY, "false"))) {
```

Comment:
Any reason we want to make this one configurable?

Reply:
I doubt we would... more an insurance policy for anyone (any data lake anywhere) , who might wish to disable e.g. in case it creates an undesirable interaction with their chosen RDBMS.  do you think I'm being too paranoid?

Reply:
yeah, I get the point that we want it to be more flexible. But I also want to slim the configurations as well unless you think that setting it to be true will be operational expansive for the jdbc and in some case we want disable it. Will leave it up to you.

Reply:
Makes sense to me